server/src/test/java/org/cloudfoundry/identity/uaa/client/ClientAdminBootstrapTests.java

```diff
@@ -562,7 +562,7 @@ void clientWithoutGrantTypeFails() {
         );
     }
 
-    private static ClientDetails doSimpleTest(
+    protected static ClientDetails doSimpleTest(
```

Comment:
It appears that the access modifier of this method and the other two methods are changed to 'protected' so they can be accessed from 'ClientAdminBootstrapMultipleSecretsUpdateTests' class, which is in the same package.  In that case, since they only need  package level access, changing them to have no access modifier would be better.